include/baresip.h

```diff
@@ -354,6 +356,18 @@ int  message_send(struct ua *ua, const char *peer, const char *msg,
 		  sip_resp_h *resph, void *arg);
 
 
+/*
+ * PUBLISH RTCP-XR
+ */
+
+typedef void (rtcpxr_recv_h)(const struct pl *peer, const struct pl *ctype,
+			      struct mbuf *body, void *arg);
+int  rtcpxr_init(rtcpxr_recv_h *recvh, void *arg);
```

Comment:
rtcpxr_init is not called from anywhere.

do you also intend for baresip to *receive* PUBLISH events ?